### DISCUSSION TITLE
Miner suggestion on music assistant icon in gui

### DISCUSSION THREAD

**Discussion by pointjalta** [music-assistant/support#2039] [Feature requests and ideas]
First of all, I installed music-assistant yesterday, I played around with it a little and so far I think it's a great improvement. An extra plus is that it can be used in home assistant but also in an independent container elsewhere in your network. Up to now, I have very little to complain about.

There is only one little thing that is putting me off a little. The side menu toggles between thin and wider view when clicking on the left pointing icon that is placed under the settings icon. It's fine but most interfaces that we use these days have a similar function to slimmer or expand a side menu. But usually it's places on the top of the interface. 

You know..., the spot where Music Assistant has its logo and name. And that isn't clickable to my modest annoyance 😥. Please make it a clickable role in the interface like that left pointing thing under settings. I think that would feel more logical for most people.

**Long story for a small suggestion. So you can image how joyfull I would be when this suggestion would be implemented by you guys. Possibly not just me alone.**

Oh... one more thing.., I love the simple no nonsense clean interface. But **where can I find the css codes so that I can theme music -assistant in line with my home-assistant themes?**

**Comment by marcelveldt** [comment]
The menu is still subject to discussion. We are slowly working towards the final design so consider this as not final yet, we had to make some concessions here and there.

About styling it to your own theme: you can't. This is NOT a Home Assistant frontend component but its own web app.
